From 8ead51bb37c1d95bdba7f224288255de95a05034 Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Mon, 19 Aug 2013 22:28:01 +0200
Subject: [PATCH 1/4] spec: Mark configuration files with %config

Explicitly declaring configuration files as such, prevents rpmlint
from falsely complaining that non-configuration files are being installed
in /etc

Signed-off-by: John Kacur <jkacur@redhat.com>
---
 rpm/SPECS/tuna.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rpm/SPECS/tuna.spec b/rpm/SPECS/tuna.spec
index 9393df6..4dd26f8 100644
--- a/rpm/SPECS/tuna.spec
+++ b/rpm/SPECS/tuna.spec
@@ -86,8 +86,8 @@ rm -rf %{buildroot}
 %{_datadir}/tuna/
 %{python_sitelib}/tuna/
 %{_mandir}/man8/tuna.8*
-%{_sysconfdir}/tuna.conf
-%{_sysconfdir}/tuna/*
+%config(noreplace) %{_sysconfdir}/tuna.conf
+%config %{_sysconfdir}/tuna/example.conf
 %{_datadir}/polkit-1/actions/org.tuna.policy
 
 %files -n oscilloscope
-- 
1.8.1.4

